src/main/com/homeworks/week8/credit/card/CreditCard.java

```
@@ -0,0 +1,43 @@
+package com.homeworks.week8.credit.card;
+
+import java.util.Objects;
+
+public class CreditCard {
+
+    private double balance;
+
+    private final String cardNumber;
+
+    public CreditCard(double balance, String cardNumber) {
+        this.balance = balance;
+        this.cardNumber = cardNumber;
+    }
+
+    public double getBalance() {
+        return balance;
+    }
+
+    public String getCardNumber() {
+        return cardNumber;
+    }
+
+    public void debit(double amount) {
```

let's throw an exception if the amount is bigger than the balance